identity-service/src/routes/healthCheck.js

```diff
@@ -171,37 +70,37 @@ module.exports = function (app) {
       // check if there have been any attempts in the time window that we processed the block health check
       const attemptedTxsInRedis = await redis.zrangebyscore(
         'relayTxAttempts',
-        minBlockTime,
-        maxBlockTime
+        '-inf',
+        'inf'
       )
       const successfulTxsInRedis = await redis.zrangebyscore(
         'relayTxSuccesses',
-        minBlockTime,
-        maxBlockTime
+        '-inf',
+        'inf'
       )
       const failureTxsInRedis = await redis.zrangebyscore(
         'relayTxFailures',
-        minBlockTime,
-        maxBlockTime
+        '-inf',
+        'inf'
       )
```

Comment:
this redis method is deprecated, should we update it to the new version?
https://redis.io/commands/zrangebyscore/